EAS energy model source: https://github.com/kdrag0n/eas_sdm660/commit/1b1da2ca5e64726b8ba36865f0e23b164f7d102c
By Danny Lin <danny@kdrag0n.dev>

Pzqqt: Add two special frequency of sdm636 (LITTLE: 1612800, big: 1804800)(by linear interpolation).
--- 
+++ 
@@ -17067,28 +17067,28 @@
 		compatible = "sched-energy";
 
 		core-cost0 {
-			busy-cost-data = <0x9ab00 0x29 0xdc500 0x46 0x10fe00 0x53 0x156300 0x92 0x177000 0x9e 0x189c00 0xb7 0x1aa900 0xe4 0x1c2000 0x11d>;
+			busy-cost-data = <0x9ab00 0x17 0xdc500 0x8b 0x10fe00 0x103 0x156300 0x1e2 0x177000 0x2b2 0x189c00 0x2f3 0x1aa900 0x366 0x1c2000 0x44e>;
 			idle-cost-data = <0x14 0x10 0x0c 0x08>;
 			linux,phandle = <0x05>;
 			phandle = <0x05>;
 		};
 
 		core-cost1 {
-			busy-cost-data = <0x10fe00 0x133 0x156300 0x1e5 0x1aa900 0x359 0x1b8a00 0x373 0x1de200 0x4c6 0x20d000 0x638 0x21b100 0x660>;
+			busy-cost-data = <0x10fe00 0x2ed 0x156300 0x5a3 0x1aa900 0xa10 0x1b8a00 0xbbe 0x1de200 0x1039 0x20d000 0x1735 0x21b100 0x1a20>;
 			idle-cost-data = <0x64 0x50 0x3c 0x28>;
 			linux,phandle = <0x0e>;
 			phandle = <0x0e>;
 		};
 
 		cluster-cost0 {
-			busy-cost-data = <0x9ab00 0x04 0xdc500 0x05 0x10fe00 0x07 0x156300 0x09 0x177000 0x09 0x189c00 0x0a 0x1aa900 0x0b 0x1c2000 0x0d>;
+			busy-cost-data = <0x9ab00 0x08 0xdc500 0x0e 0x10fe00 0x14 0x156300 0x1d 0x177000 0x23 0x189c00 0x40 0x1aa900 0x31 0x1c2000 0x3e>;
 			idle-cost-data = <0x04 0x03 0x02 0x01>;
 			linux,phandle = <0x06>;
 			phandle = <0x06>;
 		};
 
 		cluster-cost1 {
-			busy-cost-data = <0x10fe00 0x0e 0x156300 0x11 0x1aa900 0x19 0x1b8a00 0x1f 0x1de200 0x1f 0x20d000 0x25 0x21b100 0x2c>;
+			busy-cost-data = <0x10fe00 0x87 0x156300 0xc1 0x1aa900 0x110 0x1b8a00 0x129 0x1de200 0x16b 0x20d000 0x207 0x21b100 0x249>;
 			idle-cost-data = <0x04 0x03 0x02 0x01>;
 			linux,phandle = <0x0f>;
 			phandle = <0x0f>;
